src/code/sys_cfb.c

```diff
@@ -0,0 +1,125 @@
+#include "global.h"
+
+OSViMode sNotebookViMode;
```

Comment:
```suggestion
OSViMode sBombersNotebookViMode;
```
As in the Bombers Notebook?

Reply:
I think it's also used for Granny's stories. this is more just a placeholder name